app/models/rubric_criterion.rb

```diff
@@ -25,6 +25,7 @@ class RubricCriterion < Criterion
 
   has_many :test_groups, as: :criterion
 
+  validate :only_update_if_results_unreleased
```

Comment:
This is good, but you should actually move it into the `Criterion` superclass so that it affects all three criteria types. And you can add tests for the other two as well.